{"uid":"33f0a8cc279ffa96","name":"Login with valid credentials test","fullName":"tests.login_test.TestLogin#test_valid_login","historyId":"18657c207d70aa57845e0b7f4751c290","time":{"start":1702776034377,"stop":1702776044577,"duration":10200},"description":"valid login","descriptionHtml":"<p>valid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x558b3d2d2d33 <unknown>\n#1 0x558b3cf8ff87 <unknown>\n#2 0x558b3cfdfc0e <unknown>\n#3 0x558b3cfdfd11 <unknown>\n#4 0x558b3d02b004 <unknown>\n#5 0x558b3d00936d <unknown>\n#6 0x558b3d028297 <unknown>\n#7 0x558b3d0090e3 <unknown>\n#8 0x558b3cfd1044 <unknown>\n#9 0x558b3cfd244e <unknown>\n#10 0x558b3d297861 <unknown>\n#11 0x558b3d29b785 <unknown>\n#12 0x558b3d285285 <unknown>\n#13 0x558b3d29c41f <unknown>\n#14 0x558b3d26920f <unknown>\n#15 0x558b3d2c0028 <unknown>\n#16 0x558b3d2c01f7 <unknown>\n#17 0x558b3d2d1ed4 <unknown>\n#18 0x7fe310094ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7febb0944f50>\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"valid login\")\n    @allure.title(\"Login with valid credentials test\")\n    @allure.tag(\"Tagged test\")\n    @pytest.mark.flaky(reruns=1)\n    def test_valid_login(self, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:41: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"ee9723eb2c82cf54540ab72c9740ceff\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7febb0a01580>\nmessage = ''\n\n    def until(self, method: Callable[[WebDriver], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x558b3d2d2d33 <unknown>\nE       #1 0x558b3cf8ff87 <unknown>\nE       #2 0x558b3cfdfc0e <unknown>\nE       #3 0x558b3cfdfd11 <unknown>\nE       #4 0x558b3d02b004 <unknown>\nE       #5 0x558b3d00936d <unknown>\nE       #6 0x558b3d028297 <unknown>\nE       #7 0x558b3d0090e3 <unknown>\nE       #8 0x558b3cfd1044 <unknown>\nE       #9 0x558b3cfd244e <unknown>\nE       #10 0x558b3d297861 <unknown>\nE       #11 0x558b3d29b785 <unknown>\nE       #12 0x558b3d285285 <unknown>\nE       #13 0x558b3d29c41f <unknown>\nE       #14 0x558b3d26920f <unknown>\nE       #15 0x558b3d2c0028 <unknown>\nE       #16 0x558b3d2c01f7 <unknown>\nE       #17 0x558b3d2d1ed4 <unknown>\nE       #18 0x7fe310094ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:101: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[{"name":"json_data","time":{"start":1702775849748,"stop":1702775849748,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1702775849748,"stop":1702775849748,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1702775849748,"stop":1702775849748,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"valid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x558b3d2d2d33 <unknown>\n#1 0x558b3cf8ff87 <unknown>\n#2 0x558b3cfdfc0e <unknown>\n#3 0x558b3cfdfd11 <unknown>\n#4 0x558b3d02b004 <unknown>\n#5 0x558b3d00936d <unknown>\n#6 0x558b3d028297 <unknown>\n#7 0x558b3d0090e3 <unknown>\n#8 0x558b3cfd1044 <unknown>\n#9 0x558b3cfd244e <unknown>\n#10 0x558b3d297861 <unknown>\n#11 0x558b3d29b785 <unknown>\n#12 0x558b3d285285 <unknown>\n#13 0x558b3d29c41f <unknown>\n#14 0x558b3d26920f <unknown>\n#15 0x558b3d2c0028 <unknown>\n#16 0x558b3d2c01f7 <unknown>\n#17 0x558b3d2d1ed4 <unknown>\n#18 0x7fe310094ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7febb0944f50>\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"valid login\")\n    @allure.title(\"Login with valid credentials test\")\n    @allure.tag(\"Tagged test\")\n    @pytest.mark.flaky(reruns=1)\n    def test_valid_login(self, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:41: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"ee9723eb2c82cf54540ab72c9740ceff\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7febb0a01580>\nmessage = ''\n\n    def until(self, method: Callable[[WebDriver], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x558b3d2d2d33 <unknown>\nE       #1 0x558b3cf8ff87 <unknown>\nE       #2 0x558b3cfdfc0e <unknown>\nE       #3 0x558b3cfdfd11 <unknown>\nE       #4 0x558b3d02b004 <unknown>\nE       #5 0x558b3d00936d <unknown>\nE       #6 0x558b3d028297 <unknown>\nE       #7 0x558b3d0090e3 <unknown>\nE       #8 0x558b3cfd1044 <unknown>\nE       #9 0x558b3cfd244e <unknown>\nE       #10 0x558b3d297861 <unknown>\nE       #11 0x558b3d29b785 <unknown>\nE       #12 0x558b3d285285 <unknown>\nE       #13 0x558b3d29c41f <unknown>\nE       #14 0x558b3d26920f <unknown>\nE       #15 0x558b3d2c0028 <unknown>\nE       #16 0x558b3d2c01f7 <unknown>\nE       #17 0x558b3d2d1ed4 <unknown>\nE       #18 0x7fe310094ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:101: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1702776034377,"stop":1702776044577,"duration":10200},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x558b3d2d2d33 <unknown>\n#1 0x558b3cf8ff87 <unknown>\n#2 0x558b3cfdfc0e <unknown>\n#3 0x558b3cfdfd11 <unknown>\n#4 0x558b3d02b004 <unknown>\n#5 0x558b3d00936d <unknown>\n#6 0x558b3d028297 <unknown>\n#7 0x558b3d0090e3 <unknown>\n#8 0x558b3cfd1044 <unknown>\n#9 0x558b3cfd244e <unknown>\n#10 0x558b3d297861 <unknown>\n#11 0x558b3d29b785 <unknown>\n#12 0x558b3d285285 <unknown>\n#13 0x558b3d29c41f <unknown>\n#14 0x558b3d26920f <unknown>\n#15 0x558b3d2c0028 <unknown>\n#16 0x558b3d2c01f7 <unknown>\n#17 0x558b3d2d1ed4 <unknown>\n#18 0x7fe310094ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 38, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 101, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"e21a0c459fade366","name":"public ip address","source":"e21a0c459fade366.txt","type":"text/plain","size":11},{"uid":"837dffafaf022812","name":"Cookies","source":"837dffafaf022812.json","type":"application/json","size":3361},{"uid":"7ddda061b8cc1a43","name":"Session Storage","source":"7ddda061b8cc1a43.json","type":"application/json","size":358},{"uid":"93627424d72502ac","name":"Local Storage","source":"93627424d72502ac.json","type":"application/json","size":208},{"uid":"aa8fa916a57afcc4","name":"Console Logs","source":"aa8fa916a57afcc4.json","type":"application/json","size":2},{"uid":"b81f52dd51599bc0","name":"Full Page Screenshot","source":"b81f52dd51599bc0.png","type":"image/png","size":1282255},{"uid":"e3d5fb51e101eacb","name":"URL","source":"e3d5fb51e101eacb.uri","type":"text/uri-list","size":23},{"uid":"d9522755df989dd0","name":"Network Logs","source":"d9522755df989dd0.json","type":"application/json","size":25884}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"tag","value":"Tagged test"},{"name":"feature","value":"Login"},{"name":"epic","value":"Security"},{"name":"severity","value":"blocker"},{"name":"tag","value":"flaky(reruns=1)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"login_test"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az774-90"},{"name":"thread","value":"2348-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.login_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[{"uid":"4bc9b5ed69a1e80b","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x557469748d33 <unknown>\n#1 0x557469405f87 <unknown>\n#2 0x557469455c0e <unknown>\n#3 0x557469455d11 <unknown>\n#4 0x5574694a1004 <unknown>\n#5 0x55746947f36d <unknown>\n#6 0x55746949e297 <unknown>\n#7 0x55746947f0e3 <unknown>\n#8 0x557469447044 <unknown>\n#9 0x55746944844e <unknown>\n#10 0x55746970d861 <unknown>\n#11 0x557469711785 <unknown>\n#12 0x5574696fb285 <unknown>\n#13 0x55746971241f <unknown>\n#14 0x5574696df20f <unknown>\n#15 0x557469736028 <unknown>\n#16 0x5574697361f7 <unknown>\n#17 0x557469747ed4 <unknown>\n#18 0x7faada694ac3 <unknown>","time":{"start":1702776021276,"stop":1702776031480,"duration":10204}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["security","Tagged test","flaky(reruns=1)"]},"source":"33f0a8cc279ffa96.json","parameterValues":[]}